apps/system/utils/netcmd.c

```diff
@@ -548,9 +567,12 @@ const static tash_cmdlist_t net_utilcmds[] = {
 	{"ifconfig", cmd_ifconfig, TASH_EXECMD_SYNC},
 	{"ifdown", cmd_ifdown, TASH_EXECMD_SYNC},
 	{"ifup", cmd_ifup, TASH_EXECMD_SYNC},
-#ifdef NET_LWIP_STATS_DISPLAY
+#ifdef LWIP_STATS_DISPLAY
```

Comment:
LWIP is just the name of IP stack we have chosen, not a general term.
So 'lwip' term needs to be hidden inside lwip stack layer, not to be exposed to app developers.
my recommendation is net_stats instead of lwip_stats.

Reply:
I agree to your comment. LWIP_STATS_DISPLAY is changed to CONFIG_NET_STATS_DISPLAY.